understanding/20/time-based-media.html

```diff
@@ -7,106 +7,79 @@
 </head>
 <body>
    <h1>Understanding Time-Based Media</h1>
-   
-   
    <section id="intent">
       <h2>Intent of Time-Based Media</h2>
-      
-      
-      <p>The purpose of this guideline is to provide access to time-based and synchronized
-         media. This includes media that is:
+      <p>
+        The purpose of this guideline is to provide access to time-based and synchronized media. 
+        This includes media that is:
       </p>
-      
       <ul>
-         
-         <li>audio-only</li>
-         
-         <li>video-only </li>
-         
-         <li>audio-video </li>
-         
-         <li>audio and/or video combined with interaction </li>
-         
+         <li>audio-only,</li>
```

Comment:
The addition of punctuation and conjunctions to these list items goes against the [WAI Style Guide for lists](https://www.w3.org/wiki/WAI_Style_Guide#Lists).